operator/webhook/addons/certmanager/main.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import sys
+import os
+from typing import Mapping, List
+
+_LOGGER = logging.getLogger(__name__)
+
+from webhook.core.sync import sync
+
+
+def _new_certificate(object) -> Mapping:
+    metadata = object.get("metadata", None)
+    if metadata is None:
+        _LOGGER.error("IntegrationRoute does not contain metadata.")
+        return {}
+
+    annotations = metadata.get("annotations", None)
+    if annotations is None:
+        _LOGGER.error("IntegrationRoute does not contain metadata.annotations.")

```

Comment:
:question: Should this be logged as an error? It should be acceptable for an integration-route not to require a cert. How about a `debug` log that states there is nothing to be done instead?

Reply:
Changed